SPECS/valgrind/valgrind.spec

```diff
@@ -2,14 +2,16 @@
 Summary:        Memory Management Debugger.
 Name:           valgrind
 Version:        3.22.0
-Release:        1%{?dist}
+Release:        2%{?dist}
 License:        GPL-2.0-or-later
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Debuggers
 URL:            https://valgrind.org
 Source0:        https://sourceware.org/pub/%{name}/%{name}-%{version}.tar.bz2
-Requires:       glibc-debuginfo
```

Comment:
Adding for context but an alternative to this patch seems like it'd be to compile glibc without `-fno-builtin-strlen` which seems riskier